__tests__/app.test.js

```diff
@@ -48,3 +48,38 @@ describe('/api', () => {
       });
   });
 });
+
+describe('/api/articles/:article_id', () => {
+  test('GET: 200 - Responds with the requested article if it exists', () => {
+    return request(app)
+      .get('/api/articles/2')
+      .expect(200)
+      .then(({ body: {article}}) => {
+        expect(article).toHaveProperty("article_id");
+        expect(article).toHaveProperty("title");
+        expect(article).toHaveProperty("topic");
+        expect(article).toHaveProperty("author");
+        expect(article).toHaveProperty("created_at");
+        expect(article).toHaveProperty("votes");
+        expect(article).toHaveProperty("article_img_url");
+        expect(article).toHaveProperty("body");
```

Comment:
You could test for the exact values here of the articles object you’ve requested, especially the article_id. This would make your test more robust!